firmware/application/ui_navigation.cpp

```diff
@@ -555,6 +560,76 @@ void SystemStatusView::rtc_battery_workaround() {
     }
 }
 
+void SystemStatusView::new_sdcard_structure_checker() {
+    const std::filesystem::path root_dir = u"/";
+    const std::filesystem::path resources_dir = u"SYS";
```

Comment:
It's a little bit sad about this part, because the return value of scan_root_files method are returning the path without first `/`. SO it's still different than std path.
And that methods is widely used in fileman app, which needing more work and test to get it using std path format. 

I made others globally tho.